components/ClaimDetails.tsx

```diff
@@ -1,34 +1,25 @@
 import { useTranslation } from 'next-i18next'
 import { InfoField } from './InfoField'
+import { ClaimDetailsContent } from '../types/common'
 
-export interface ClaimDetailsProps {
+export interface ClaimDetailsProps extends ClaimDetailsContent {
```

Comment:
Note: I used [`extends`](https://www.typescriptlang.org/docs/handbook/2/objects.html#extending-types) on a number of component interfaces so that I could have an interface that was content-only (e.g. `ClaimDetailsContent`) and then have an interface that contained the content as well as the additional props, such as `loading (e.g. `ClaimDetailsProps`). 

Alternatively, if this is hard to read, we can also use nested props, like this:

```tyescript
export interface ClaimDetailsProps {
  loading: boolean
  content: ClaimDetailsContent
}
```
but I thought it was less noisy to have the props on the same level